splunk_event/conf.yaml.example

```diff
@@ -48,9 +48,36 @@ init_config:
 # It is possible to specify multiple saved_searches on a single instance.
 instances:
   - url: "https://localhost:8089"
+
+    # username: "admin" # deprecated
+    # password: "admin" # deprecated
+
     # verify_ssl_certificate: false
-    username: "admin"
-    password: "admin"
+
+    ## new authentication structure either token or basic auth will be used.
```

Comment:
Suggestion: "Integration supports either basic authentication or token based authentication. Token based authentication is preferred before basic authentication." ?